src/Compute/Compute.Sdk/README.md

```diff
@@ -0,0 +1,81 @@
+# Overall
+This directory contains the service clients of Az.Compute module.
+
+## Run Generation
+In this directory, run AutoRest:
+```
+autorest.cmd README.md --version=v2
+```
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+csharp: true
+clear-output-folder: true
+reflect-api-versions: true
+openapi-type: arm
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+
+title: ComputeManagementClient
+payload-flattening-threshold: 1
+
+directive:
+    # dynamically add a DummyOrchestrationServiceName value to the enum 
+  - from: virtualMachineScaleSet.json
+    where: $..enum
+    transform: >-
+      if( $.length === 1 && $[0] === "AutomaticRepairs") { 
+        $.push('DummyOrchestrationServiceName');
+      }
+      return $;
+    
+    # remove it from the C# generated code
+  - from: source-file-csharp
+    where: $ 
+    transform: >-
+      return $.
+        replace(/.*public const string DummyOrchestrationServiceName.*/g,'').
+        replace(/, 'DummyOrchestrationServiceName'/g,'');
+  - from: gallery.json
+    where: $.definitions.GalleryTargetExtendedLocation.properties.storageAccountType["x-ms-enum"].name
+    transform: return "EdgeZoneStorageAccountType"
+```
+
+
+### 
+``` yaml 
+input-file:
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json
```

Comment:
We have updated the format of input-file like the following because all the JSON files must be under the same commit id.
```suggestion
``` yaml
commit: {commit} # Replace with a real commit id
input-file:
  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json
  ...
```